applications/nrf_desktop/src/hw_interface/Kconfig.motion

```diff
@@ -80,7 +80,7 @@ config DESKTOP_MOTION_RIGHT_KEY_ID
 	  ID of key use for generating right motion.
 
 config DESKTOP_MOTION_SIMULATED_EDGE_TIME
-	int "Time for transition between two points in a trajectory [ms]"
+	int "Time for transition between two points in a trajectory [us]"
 	depends on DESKTOP_MOTION_SIMULATED_ENABLE
 	range 8 1000000
 	default 16
```

Comment:
shouldn't we adjust the range and defaults here and multiply them by 1000?

Reply:
increased default value to `16ms` aligned to `2^x` and adjusted higher range by `* 1000`  so it is as big as it was before